site2/tools/build-site.sh

```diff
@@ -22,10 +22,12 @@ ROOT_DIR=$(git rev-parse --show-toplevel)
 VERSION=$(${ROOT_DIR}/src/get-project-version.py)
 
 NEXT=$1
-WEBSITE_DIR="website"
+WEBSITE_DIR=${ROOT_DIR}/site2/website
 if [ -n "$NEXT" ]; then
-  WEBSITE_DIR="website-"$NEXT
+  WEBSITE_DIR=${ROOT_DIR}/site2/website-$NEXT
 fi
+TOOLS_DIR=${ROOT_DIR}/site2/tools
+GEN_SITE_DIR=$GEN_SITE_DIR
```

Comment:
why `GEN_SITE_DIR=$GEN_SITE_DIR`? Is this redundant? @SignorMercurio

Reply:
This is a mistake. I'll fix it and try to test it locally.